trajoptlib/include/trajopt/constraint/Constraint.hpp

```diff
@@ -61,6 +62,7 @@ static_assert(ConstraintType<PointPointMaxConstraint>);
 static_assert(ConstraintType<PoseEqualityConstraint>);
 static_assert(ConstraintType<TranslationEqualityConstraint>);
 static_assert(ConstraintType<PointLineRegionConstraint>);
+static_assert(ConstraintType<LaneConstraint>);
```

Comment:
Could you sort these alphabetically while you're here? It will match the header include order.